python/test/unit/language/test_tuple.py

```diff
@@ -162,3 +162,76 @@ def mul(x, a):
     ty = Tensor(y, y.shape, y.stride())
     _namedtuple_kernel[(1, )](function, tx, ty, 64, 64)
     assert torch.allclose(y, x[:16, :16] * a)
+
+
+@pytest.mark.interpreter
+def test_eq(device):
```

Comment:
our linter is drunk lol

Reply:
Let me see if I can make it look better...